theme/templates/resource-landing-page/modals.html

```diff
@@ -435,14 +435,14 @@ <h4 class="modal-title" id="submit-for-publication-title">Submit resource for pu
             <div class="modal-body">
                 <p>
                     Upon submitting the resource for publication it will be permanently published and assigned a
-                    digital object identifier (DOI). This is <strong>final</strong> and changes can only be made
+                    minimal identifier (MINID). This is <strong>final</strong> and changes can only be made
                     on a new version (copy) of the resource.
                 </p>
                 <p>
                      Please confirm that you accept the CommonsShare Publication Agreement shown below.
                 </p>
                 <div class="iframe-border">
-                    <iframe id="pub-agreement-frame" scrolling="no" src="https://www.hydroshare.org/publication-agreement/"></iframe>
+                    <iframe id="pub-agreement-frame" scrolling="no" src="https://park.commonsshare.org/publication-agreement/"></iframe>
```

Comment:
We should not hardcode park.commonsshare.org here. We can use www.commonsshare.org instead.